test/controllers/NCTSMessageControllerSpec.scala

```diff
@@ -334,6 +342,7 @@ class NCTSMessageControllerSpec extends SpecBase with ScalaCheckPropertyChecks w
 
           val result = route(application, request).value
 
+          contentAsString(result) mustEqual "External Submission Failure Future(Success(SubmissionFailureExternal))"
```

Comment:
The code that generates this error message still needs to be amended. The error message should be removed.